wagtail/images/models.py

```diff
@@ -97,10 +97,15 @@ def is_stored_locally(self):
     def get_file_size(self):
         if self.file_size is None:
             try:
-                self.file_size = self.file.size
-            except OSError:
-                # File doesn't exist
-                return
+                size = self.file.size
+            except Exception:
```

Comment:
When doing catch-all except blocks like this, we must add the following block just before in order for debugging tools to work correctly:

```python
except (KeyboardInterrupt, SystemExit):
    raise
except:
    ...
```